RPGManager_development/src/rpg_database/character_sheet/Shields.java

```diff
@@ -0,0 +1,38 @@
+package rpg_database.character_sheet;

```

Comment:
Ez abszolút nem volt része a feladatnak, és bár tök fasza hogy szántál rá időt, de mivel elég sok energiát elvesz a review, nem reviewznék le még egy komplett classt és a tesztjeit teljesen feleslegesen. Ezt gondolom te is megérted. Szerintem erről beszéljünk a szerdai meetingen hogy mi legyen vele, legyen -e rá új task, van -e rá szükség, hogy kerül ez a képbe egyáltalán, mert nem triviális, befolyásolja Talent is, ez Item is egyszerre meg annál több is, befolyásolja a defense-et stb. stb. stb. Addig is a helyedben (ha ki akarod pörgetni az Armor_Types-ot) akkor kiraknám lokálisan egy külön branchre, vagy elmenteném valahova későbbi használatra, mert tök jó hogy ez megvan, csak nem most jött el az ideje hogy ezt ilyen formában berakjuk, pláne úgy hogy ez fölött is tudnék fél órát-órát ücsörögni.